File: CS50/sentimental-mario-more/mario.py
from cs50 import get_int

while (True):
    try:
        n = int(get_int("Height: "))
        if n > 0 and n < 9:
            break
    except ValueError:
        continue
for i in range(1, n+1):
    block = (n-i)*' ' + i*'#'
    # print(block + '  ' + block[-1:-i-1:-1])
    print(block + '  ' + block[n-i:])

File: CS50/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //request user input for height in the range
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (int i = 1; i <= height; i++)
    {
        for (int j = height - i ; j >= 1 ; j--)
        {
            printf(" ");
        }
        for (int j = 0 ; j < i ; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0 ; j < i ; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

